src/lib.rs

```diff
@@ -59,12 +59,14 @@ impl Bencher {
     ///
     /// # Timing loop
     ///
-    /// ``` ignore
-    /// start = Instant::new();
+    /// ```rust,no_run
```

Comment:
It's just Markdown's syntax to set the language used in the code block (just like `text` below). I like to add it to make things more explicit. The `ignore` or `no_run` flags only work when `rust` is set of course – rustdoc assumes `rust` by default when no language is set.